scripts/needed-auto-tfvars-files/storage/storage.auto.tfvars.example

```diff
@@ -1,8 +1,13 @@
-storage_accounts = {
+storage_accounts = { # storage account
   adminsvc1 = {
     delete_protection         = false //Set to false to allow deletion
     prefix_name               = "adminsvc1"
     storage_type              = "azurefiles"
+    #authorized_ip_ranges = { anyone = "97.118.251.104" }
```

Comment:
Please delete commented-out code here and elsewhere.

Reply:
Fixed